### PR TITLE
docs: Add readme file

### PR DESCRIPTION
**Pull request by jukkar**
Add simple documentation how to deploy nri-resmgr in a cluster node.

Signed-off-by: Jukka Rissanen <jukka.rissanen@linux.intel.com>

### CODE REVIEW

**Comment by kad** [comment]
looks good, maybe just adding some few lines about "now let's run some test pod and see that pinning is happened"?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   # echo "disableConnections: false" > /etc/nri/nri.conf
+   # mkdir -p /opt/nri/plugins
+```
+
+Note that containerd must have NRI support enabled and NRI is currently only
+available in 1.7beta or later containerd release. This is why you must do
+some extra steps in order to enable NRI plugin support in containerd.
+
+Edit the `/etc/containerd/config.toml` file and set `plugins."io.containerd.nri.v1.nri"`
+option `disable = false` and restart containerd.
```
Comment (kad) [inline README.md]: better show snippet of config.toml on how it is done.

**Comment by jukkar** [comment]
@kad updated, PTAL

File: README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+```
+
+Edit the `/etc/containerd/config.toml` file and set `plugins."io.containerd.nri.v1.nri"`
+option `disable = true` to `disable = false` and restart containerd.
+
+Then deploy NRI resource manager plugin
+
+```
+   $ ctr -n k8s.io images import nri-resmgr-topology-aware-image-9797e8de7107.tar
+   $ kubectl apply -f nri-resmgr-topology-aware-deployment.yaml
```
Comment (klihub) [inline README.md:64]: We need also the adjustment CRD...
$ kubectl apply -f path//to/adjustment-schema.yaml

**Comment by jukkar** [comment]
Already applied

**Pull request closed without merging**